{"origin": "http://10.8.1.3:8080", "type": "message", "message": "ResponseStatusError: Response 404: HTML content", "error": "ResponseStatusError", "provider": {"name": "Goabror", "url": "https://goabror.uz", "label": null, "model": "gpt-4"}}
{"origin": "http://10.8.1.3:8080", "type": "message", "message": "ValueError: Unexpected response format: {'status': 'error', 'msg': 'Estamos verificando autom\u00e1ticamente si eres un humano. Por favor, int\u00e9ntalo de nuevo en 1 o 2 segundos.'}", "error": "ValueError", "provider": {"name": "ChatGptEs", "url": "https://chatgpt.es", "label": null, "model": "gpt-4"}}
{"origin": "http://10.8.1.3:8080", "type": "error", "error": "RateLimitError", "message": "RateLimitError: You have reached your request limit for the hour. [Upgrade for higher rate limits](https://www.blackbox.ai/pricing?ref=rate-limit)", "provider": {"name": "Blackbox", "url": "https://www.blackbox.ai", "label": "Blackbox AI", "model": "DeepSeek-R1"}}
{"origin": "http://10.8.1.3:8080", "type": "error", "error": "RateLimitError", "message": "RateLimitError: You have reached your request limit for the hour. [Upgrade for higher rate limits](https://www.blackbox.ai/pricing?ref=rate-limit)", "provider": {"name": "Blackbox", "url": "https://www.blackbox.ai", "label": "Blackbox AI", "model": "gpt-4o-mini"}}
{"origin": "http://10.8.1.3:8080", "type": "message", "message": "DuckDuckGoChallengeError: Challenge error after 3 retries: Challenge error after 3 retries", "error": "DuckDuckGoChallengeError", "provider": {"name": "DDG", "url": "https://duckduckgo.com/aichat", "label": "DuckDuckGo AI Chat", "model": "gpt-4"}}
